subsys/caf/modules/power_manager.c

```diff
@@ -222,6 +267,33 @@ static bool event_handler(const struct event_header *eh)
 		EVENT_SUBMIT(event);
 		return false;
 	}
+#if IS_ENABLED(CONFIG_CAF_POWER_MANAGER_RUNTIME_TIMEOUT_SET)
+	if (is_power_manager_configure_timeout_event(eh)) {
+		int err;
```

Comment:
please move to the tightest scope. I know this will repeat err declaration but err is unused really

when you declare variables try to put the ones with shortest names at the end - makes the text block neater to look at in my opinion